labelCloud/io/labels/kitti.py

```diff
@@ -89,3 +113,34 @@ def export_labels(self, bboxes: List[BBox], pcd_path: Path) -> None:
             f"Exported {len(bboxes)} labels to {path_to_file} "
             f"in {self.__class__.__name__} formatting!"
         )
+
```

Comment:
Please also add the reverse transformation for writing the labels in the `export_labels` function above.

- [ ] reverse transformation in write function

Reply:
Done